lib/Caxy/HtmlDiff/HtmlDiff.php

```diff
@@ -86,6 +86,11 @@ public function getInsertSpaceInReplace()
         return $this->config->isInsertSpaceInReplace();
     }
 
+    public function clearContent()
+    {
+      $this->content = '';
```

Comment:
The `$content` property was actually moved from the `HtmlDiff` class to the new abstract class `AbstractDiff` (as of version 0.0.6). 

So this should probably be added to the `AbstractDiff` instead.

Also, for code styling we follow the PSR-1 and PSR-2 standards (although a lot of the original code is not up to the standard yet). So if you could indent by 4 spaces instead of 2, that would be very helpful!